### ISSUE TITLE
Support for V3 Signatures

### ISSUE THREAD

**Issue by vanitasvitae** [vanitasvitae/pgpainless#361]
Hi,

I'm trying to decrypt a file that was encrypted by SecureZIP and I'm getting the following error:

```
org.pgpainless.exception.SignatureValidationException: Signature is not valid: Malformed signature. Signature has no signature creation time subpacket in its hashed area.
```

I'm aware that RFC4880 requires the presence of the signature creation time _as a hashed subpacket_. It seems a little odd to me that a commercial product like SecureZIP would make such a mistake. 

`gpg` doesn't really seem to care that there is no such subpacket, it decrypts the file just fine and states that the signature is correct (event with the `--rfc4880` flag!).

I'm trying to determine what my options are. I think I could check if the `OpenPgpMetadata` contains an (invalid) signature made with the expected key, and simply ignore this specific kind of error.

On the other hand, getting SecureZIP to fix their bug (if it's really a bug on their side) seems like a really long shot.

What do you PGP experts think?

_Originally posted by @bjansen in https://github.com/pgpainless/pgpainless/discussions/357_

**Comment by vanitasvitae** [comment]
Comment by @bjansen:

> Wait I think I got it. pgpainless throws an error on a signature packet in [version **3**](https://www.rfc-editor.org/rfc/rfc4880#section-5.2.2), but apparently hashed creation times are only required in [version **4**](https://www.rfc-editor.org/rfc/rfc4880#section-5.2.3.4).

**Comment by vanitasvitae** [comment]
I just released PGPainless 1.4.4 which contains the fix you proposed with #358

**Comment by bjansen** [comment]
Thanks a lot!
